src/pages/Statistics/Statistics.tsx

```diff
@@ -13,21 +31,313 @@ export default function Statistics() {
   const [startDate, setStartDate] = useState(location?.state?.startDate ?? "");
   const [endDate, setEndDate] = useState(location?.state?.endDate ?? "");
 
+  const settingsNames = user!.settings.map((e) => e.key);
+  const [radioValue, setRadioValue] = useState(2);
+  const [graphTitle, setGraphTitle] = useState("Weekly Happiness");
+  const [graphSubTitle, setGraphSubTitle] = useState("");
+  const [viewingEntry, setViewingEntry] = useState<Happiness | undefined>(
+    undefined,
+  );
+  const [calCollapsed, setCalCollapsed] = useState<boolean>(false);
+  const [start, setStart] = useState(
+    new Date(
+      new Date().getFullYear(),
+      new Date().getMonth(),
+      new Date().getDate() - 7,
+    ),
+  );
+  const [end, setEnd] = useState(
+    new Date(
```

Comment:
Can you not just use an empty date constructor here?

Reply:
I wanted to avoid Javascript date issues here